### PR TITLE
Move node packages to dependancies

### PR DESCRIPTION
**Pull request by jericson-edb**
## What Changed?

 I just moved some node packages from `devDependencies` to `dependencies`. I think it fixes [the problem with scripts/source/livecompare.js](https://github.com/EnterpriseDB/docs/runs/4968354738?check_suite_focus=true).

### CODE REVIEW

**Review by josh-heyer — APPROVED** [review]

**Review by josh-heyer — CHANGES_REQUESTED** [review]
This won't work; check out how it's done for CNP

**Comment by jericson-edb** [comment]
> This won't work; check out how it's done for CNP

Got it. I reset the branch (and force pushed) and tried to duplicate CNP.

File: scripts/source/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "fileprocessor",
```
Comment (josh-heyer) [inline scripts/source/package.json:2]: probably give it a different name

**Review by josh-heyer — APPROVED** [review]
at some point soon, we should roll these into fileprocessor and avoid the issue, but for now probably call the package something else. Other than that, +1

**Review by josh-heyer — APPROVED** [review]

File: scripts/source/package.json (hunk 1 above)
Comment (jericson-edb) [reply scripts/source/package.json]: Good call. And it really does make sense to unify them. I'll look at doing that next week when we tackle the other docs.

**Review by josh-heyer — APPROVED** [review]

**Pull request merged**